kubejs/server_scripts/gregtech/dense_ores.js

```diff
@@ -3,30 +3,30 @@
  */
 
 const ores = [
-    {name: 'redstone', mod: 'gtceu'},
-    {name: 'diamond', mod: 'gtceu'},
-    {name: 'emerald', mod: 'gtceu'},
-    {name: 'lapis', mod: 'gtceu'},
-    {name: 'coal', mod: 'gtceu'},
-    {name: 'nether_quartz', mod: 'gtceu'},
-    {name: 'iron', mod: 'minecraft'},
-    {name: 'gold', mod: 'minecraft'},
-    {name: 'copper', mod: 'minecraft'},
+    ['redstone', 16],
+    ['diamond', 16],
+    ['emerald', 16],
+    ['gold', 16],
+    ['lapis', 16],
+    ['iron', 16],
+    ['coal', 16],
+    ['nether_quartz', 32],
+    ['copper', 16],
 ]
 
 ServerEvents.recipes(event => {
-    ores.forEach(ore => {
-        event.recipes.gtceu.chemical_reactor(`kubejs:undensify_${ore.name}`)
-            .itemInputs(`kubejs:dense_${ore.name}_ore`)
-            .itemOutputs(`16x ${ore.mod}:raw_${ore.name}`)
+    ores.forEach([name, amount] => {
+        event.recipes.gtceu.chemical_reactor(`kubejs:undensify_${name}`)
+            .itemInputs(`kubejs:dense_${name}_ore`)
+            .itemOutputs(ChemicalHelper.get("raw", name, amount))
             .inputFluids(Fluid.of("gtceu:nitric_acid", 1000))
             .duration(200)
             .EUt(500)
     })
 
     event.recipes.gtceu.chemical_reactor("kubejs:undensify_oilsands")
         .itemInputs("kubejs:dense_oilsands_ore")
-        .itemOutputs('16x gtceu:raw_oilsands')
+        .itemOutputs('16x gtceu:endstone_oilsands_ore')
```

Comment:
I had left this untouched when I was doing the change, to allow the use of the centrifuging recipe for oilsand blocks. It allows using the miner trick for it, but oil is cheap enough that I don't think it maters.